OADP 3523 fs-backup/restic default timeout is being changed to 4 hours (#72492)
openshift/openshift-docs: modules/oadp-converting-to-new-dpa-1-2-0.adoc

# ...
----

[NOTE]
====

> The `fs-backup/restic timeout` is a one-hour default fs-backup/restic timeout. In OADP 1.3.1 the default is four hours.

Reviewer: oc explain dataprotectionapplications.oadp.openshift.io.spec.configuration.velero.args.fs-backup-timeout

KIND:     DataProtectionApplication
VERSION:  oadp.openshift.io/v1alpha1

FIELD:    fs-backup-timeout <integer>

DESCRIPTION:
     How long pod volume file system backups/restores should be allowed to run
     before timing out. (default 4h0m0s)
     
the default value for dataprotectionapplications.oadp.openshift.io.spec.configuration.velero.args.fs-backup-timeout t is Actually for both restic and kopia (in 1.3.x)

----

. Wait for the DPA to reconcile successfully.

[NOTE]
====

> The `fs-backup/restic timeout` is a one-hour default fs-backup/restic timeout. In OADP 1.3.1 the default is four hours.

Reviewer: ```suggestion
The `fs-backup/restic timeout` is one-hour by  default. In OADP 1.3.1 the default was changed to  four hours for both restic and kopia fs-backup
```

----

. Wait for the DPA to reconcile successfully.

[NOTE]
====

> The `fs-backup/restic` timeout is one-hour by default. In OADP 1.3.1 the default is four hours for both Restic and Kopia file system backup.

Reviewer 2: 1. If the timeout value is being changed for 1.3.1+ versions then  for clarity "In OADP 1.3.1 **and later**" can be used IMO.
2. Instead of using the `fs-backup/restic` parameter, can we just use "Restic"? Because in the next sentence we are using file system backup names (Rustic and Kopia) instead of parameters.
3. The default timeout value for Kopia earlier OADP 1.3.1 was one hour, like Restic? If so, then it can be mentioned in the first sentence.

> The `fs-backup/restic` timeout is one-hour by default. In OADP 1.3.1 the default is four hours for both Restic and Kopia file system backup.

Author: @xenolinux 
Thank you for your good suggestions.
